docs/proposals/compute-resource-metrics-api.md

```diff
@@ -35,15 +35,31 @@ Documentation for other releases can be found at
 
 ## Goals
 
-Provide resource usage metrics on pods and nodes on the API server to be used
-by the scheduler to improve job placement, utilization, etc. and by end users
-to understand the resource utilization of their jobs. Horizontal and vertical
-auto-scaling are also near-term uses.
+Provide resource usage metrics on pods and nodes through the API server to be
+used by the scheduler to improve job placement, utilization, etc. and by end
+users to understand the resource utilization of their jobs. Horizontal and
+vertical auto-scaling are also near-term uses. Additionally, a subset of the
```

Comment:
Thinking more about this, I'm worried that this will get confusing, especially if the API server and kubelet APIs diverge any more. I almost wonder if we want separate api groups for each (e.g. `kubelet-metrics`, or maybe just `kubelet` or `node`). On the one hand, we want consistency so keeping them together makes sense, on the other hand it should be clear which is served by which component. Separate groups also prevents reuse of common types across the APIs. Another option would be to prefix kubelet-only types with "kubelet" (e.g. s/pods/kubeletPods/), but this could get messy if we decide to start serving them at the cluster level.

Thoughts?